modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -268,9 +269,10 @@ const move = (state: Singleton.Value<State>, editor: Editor) => {
     }
 
     if (state.dragging) {
+      const mouseEventOriginatedFromWithinTheEditor = e.currentTarget === editor.getDoc().firstElementChild;
```

Comment:
Clearer to know what the first element it could in theory have been the head element.

```suggestion
      const mouseEventOriginatedFromWithinTheEditor = e.currentTarget === editor.getDoc().documentElement;
```

Reply:
This won't work for inline editors. The events are then bound on the editor.getBody() does this drag/drop overrides work with inline mode should we branch out of it in inline mode.